### PR TITLE
[plugin.audio.rne] v1.0.13

### PR DESCRIPTION
**Pull request by jamontes**
### Description
Hi,

This is a maintenance PR for the RNE add-on due to website changes:

### Changelog: 1.0.13 (2020.05.01)
- Several code fixes due to website changes.

Please, bear in mind that this is a pure Python2 version for Helix branch up to Leia.
I will summit another PR for the Matrix branch which is pure Python3.

Many thanks in advance for your help and best regards,
jamontes.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.rne/addon.xml [hunk 1]
```diff
@@ -20,6 +20,6 @@
         <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```
Comment (enen92) [inline plugin.audio.rne/addon.xml:20]: GPL-3.0-only or GPL-3.0-or-later depending on your license (SPDX identifier please). See https://spdx.org/licenses/

File: plugin.audio.rne/resources/lib/lutil.py [hunk 2]
```diff
@@ -29,50 +29,59 @@
...
-    debug_enable = debug_flag in ("true", True)
 
+def local_log(message):
+    """This function logs the messages into the main KODI log file. Called from the libraries module by other functions."""
 
-def log(message):
-    """This function logs the messages into the main KODI log file. Called from main plugin module."""
     if debug_enable:
-        print "%s" % message
+        print("%s" % message)
```
Comment (enen92) [inline plugin.audio.rne/resources/lib/lutil.py:37]: xbmc.log()

File: plugin.audio.rne/addon.xml (hunk 1 above)
Comment (jamontes) [reply plugin.audio.rne/addon.xml:20]: OK, I'll fix it. Thanks for the advice.

File: plugin.audio.rne/resources/lib/lutil.py (hunk 2 above)
Comment (jamontes) [reply plugin.audio.rne/resources/lib/lutil.py:37]: When the add-on is run under Kodi environment, it always use xbmc.log, and only if the debug flag is activated. This debug function is only invoked under development, out of the Kodi runtime environment. This behaviour is similar to other frameworks, as xbmcswift and others, and eases a lot in the task to debug and find the broken parts whenever the website changes its internal structure. So please, consider to leave this debug function as far as it is never invoked from Kodi.
It is true that the function explanation header leads to confusion as it was originally (Eden) used to log the messages into the Kodi log file, but it is no longer like that. I will modify the function explanation to adapt it to their current purpose.

**Pull request merged**